commands/doit.go

```diff
@@ -44,49 +44,32 @@ var DoitCmd = &Command{
 	},
 }
 
-// Context holds the current auth context
-var Context string
-
-// APIURL holds the API URL to use.
-var APIURL string
-
-// Token holds the global authorization token.
-var Token string
-
-// Output holds the global output format.
-var Output string
-
-// Verbose toggles verbose output.
-var Verbose bool
-
-var requiredColor = color.New(color.Bold).SprintfFunc()
-
-// Writer is where output should be written to.
-var Writer = os.Stdout
-
-// Trace toggles http tracing output.
-var Trace bool
-
-// cfgFile is the location of the config file
-var cfgFile string
-
-// cfgFileWriter is the config file writer
-var cfgFileWriter = defaultConfigFileWriter
-
-// ErrNoAccessToken is an error for when there is no access token.
-var ErrNoAccessToken = errors.New("no access token has been configured")
+var (
+	APIURL           string
+	Context          string // current auth context
+	ErrNoAccessToken = errors.New("no access token has been configured")
+	Output           string // global output format
+	Token            string // global authorization token
+	Trace            bool   // toggles http tracing output.
+	Verbose          bool
+	Writer           = os.Stdout
+
+	cfgFile       string                    // full path to config file
+	cfgFileWriter = defaultConfigFileWriter // create default cfgFileWriter
+	requiredColor = color.New(color.Bold).SprintfFunc()
+)
 
 func init() {
+	defaultConfigPath := filepath.Join(configHome(), defaultConfigName)
+	DoitCmd.PersistentFlags().StringVarP(&cfgFile, "config", "c", defaultConfigPath,
+		fmt.Sprintf("config file (default is %s)", defaultConfigPath))
```

Comment:
This seems to be printing repetitive info, eg:

```
  -c, --config string         config file (default is /home/asb/.config/doctl/config.yaml) (default "/home/asb/.config/doctl/config.yaml")
```

Looks like we can simplify it to:

```suggestion
		"config file")
```

Reply:
nice catch! That led me to digging into the defaults a bit more, which led me to doing a bit more gardening....